adding cka preparation post (#287)
ippontech/blog-usa: posts/cka-preparation-2023.md

# Relevance

Kubernetes [(K8s)](https://kubernetes.io/) is the most widely used open source container orchestration platform on the market. Originally developed at Google, it is currently hosted by the Cloud Native Computing Foundation [(CNCF)](https://www.cncf.io/). Even though Kubernetes has been steadily gaining popularity for almost a decade, there are still not enough specialists with this skill set. More and more people are getting interested in this technology and Kubernetes-savvy engineers are in high demand. CNCF in collaboration with [The Linux Foundation](https://training.linuxfoundation.org/) has created Kubernetes Certification Programs to meet the needs of the market. What makes these exams so special besides the fact that they cover extremely sought-after knowledge areas, is that they take an entirely hands-on approach. There is no way to randomly guess your way into passing them. Not only do they certify theoretical knowledge, but they also validate practical experience working on K8s clusters.

Depending on their career path, engineers can choose between three kinds of certifications:

> - [CKA](https://training.linuxfoundation.org/certification/certified-kubernetes-administrator-cka/) - Certified Kubernetes Administrator 

Reviewer: Whole thing is really great... Even though blog is about CKA, it might be worth expanding on the 3 paths described and just giving a sentence or two of context to each as to why one might consider one of the alternatives.

Author: @lanefu I have added the descriptions of 3 certs. Please review.

- [CKAD](https://training.linuxfoundation.org/certification/certified-kubernetes-application-developer-ckad/) - Certified Kubernetes Application Developer 
- [CKS](https://training.linuxfoundation.org/certification/certified-kubernetes-security-specialist/) - Certified Kubernetes Security Specialist

This post offers guidance on preparation for the CKA exam which is considered the most challenging out of three Kubernetes certifications. During this test, engineers are given access to the environment where they are faced with real-world scenarios on cluster and storage management, network and application troubleshooting, etc.

# CKA Exam Overview

> Throughout the 2-hour session, exam takers need to solve 17 tasks with different levels of difficulty. These scenarios are valued according to their complexity (1%, 4%, 8%, etc) and at least 66% score is needed to pass. The work is performed on live clusters in the terminal window and access to Kubernetes documentation is available through the whole session. It is an online, proctored exam, which means that you can take it from the comfort of your home, while a proctor will be watching you to ensure that no "extra" help is used to pass this certification. The main areas of focus in the CKA certification are:

Reviewer 2: ```suggestion
Throughout the 2-hour session, exam takers need to solve 17 tasks with varying levels of difficulty. These scenarios are weighted according to their complexity (1%, 4%, 8%, etc) and a total score of at least 66% is needed to pass. The work is performed on live clusters in the terminal window and access to Kubernetes documentation is available through the whole session. It is an online, proctored exam, which means that you can take it from the comfort of your home, while a proctor will be watching you to ensure that no "extra" help is used to pass this certification. The main areas of focus in the CKA certification are:
```

- **Troubleshooting - 30%**
- **Cluster Architecture, Installation & Configuration - 25%**
- **Services & Networking - 20%**
- **Workloads & Scheduling - 15%**
- **Storage - 10%**

After around 6 weeks of training for 2 hours a day, I was able to pass CKA on the first attempt. The great thing about this exam is that a free retake is included in the certification package which puts the minds of those preparing for it more at ease.

# Recommended training materials

> After some thorough research, it was noticed that currently the majority of those aiming to pass CKA certification use the following training materials:

Reviewer 2: ```suggestion
Through thorough research, I noticed that the majority of those aiming to pass CKA certification use the following training materials:
```
